### PR TITLE
initial without previous git info

### PR DESCRIPTION
**Pull request by marikoja**
Initial work (without previous git info)

### CODE REVIEW

File: frontend/.nyc_output/out.json [hunk 1]
```diff
@@ -0,0 +1,3985 @@
+{
+  "/Users/marikoanderson/fe_v3/src/components/auth/Login.jsx": {
```
Comment (bjkeller) [inline frontend/.nyc_output/out.json:2]: this file has path from your user, which makes me wonder if this directory something that needs to be checked in?

File: frontend/src/components/User.jsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import { withRouter, Redirect} from "react-router-dom";
+import axios from "axios";
+
+const useStyles = makeStyles(theme => ({
+  menuButton: {
+    marginRight: theme.spacing(2)
+  },
+}));
+
+const User = (props) => {
```
Comment (bjkeller) [inline frontend/src/components/User.jsx:15]: Should this be `UserMenu` instead of `User`?

File: frontend/src/components/auth/Login.jsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
+import React, { useState, useEffect } from 'react';
+import Typography from '@material-ui/core/Typography'
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid'
+import { makeStyles } from '@material-ui/core/styles';
+import { useHistory, Redirect } from "react-router-dom";
+import axios from "axios";
+
+const Login = (props) => {
```
Comment (bjkeller) [inline frontend/src/components/auth/Login.jsx:10]: would it be appropriate to call this `LoginDialog`?

File: frontend/src/components/auth/Login.jsx [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+          <p>Invalid login/password combination</p>
+        }
+        <Button className={classes.button} name="submit" type="submit" >SIGN IN</Button>
+      </form>
+    </div>
+  )
+}
+export default Login;
+
+const useStyles = makeStyles((theme) => ({
```
Comment (bjkeller) [inline frontend/src/components/auth/Login.jsx:105]: please be consistent about where you place definitions. Pretty sure that in the header this was at the top of the file

File: frontend/src/components/auth/Logout.jsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect, } from 'react';
+import Button from '@material-ui/core/Button';
+import { useHistory, Redirect, } from "react-router-dom";
+import axios from "axios";
+
+export default function Logout(props) {
```
Comment (bjkeller) [inline frontend/src/components/auth/Logout.jsx:6]: maybe `LogoutDialog`?

**Review by bjkeller — CHANGES_REQUESTED** [review]
Just a couple of things – more comments than requirements:
- Would be good to name components so that what kind of UI component they are is clear. 
  Examples would be `UserMenu` instead of `User`, or `LoginDialog` instead of `Login`.
- Should the `.nyc_output` directory be in `.gitignore`?

File: frontend/src/components/auth/Login.jsx (hunk 3 above)
Comment (marikoja) [reply frontend/src/components/auth/Login.jsx:10]: Yeah that works for me.

File: frontend/src/components/auth/Logout.jsx (hunk 5 above)
Comment (marikoja) [reply frontend/src/components/auth/Logout.jsx:6]: LogoutButton is probably the most descriptive.

File: frontend/src/components/auth/Login.jsx (hunk 3 above)
Comment (marikoja) [reply frontend/src/components/auth/Login.jsx]: Sure that makes sense to me.

File: frontend/src/components/auth/Logout.jsx (hunk 5 above)
Comment (marikoja) [reply frontend/src/components/auth/Logout.jsx]: I changed to LogoutButton, its most accurate for what is rendered.

File: frontend/.nyc_output/out.json (hunk 1 above)
Comment (marikoja) [reply frontend/.nyc_output/out.json:2]: I've added this to git ignore because it is and auto generated code coverage report from cypress

Comment (marikoja) [reply frontend/.nyc_output/out.json:2]: Added to git ignore as you suggested.

**Comment by marikoja** [comment]
@bjkeller I've updated the component names and fixed a lot of the styling issues for consistency.

**Review by bjkeller — APPROVED** [review]

**Pull request merged**